### PR TITLE
Updated for new region setup

### PR DESCRIPTION
**Pull request by troy-ameigh**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by MarciaRieferJohnston — CHANGES_REQUESTED** [review]
@troy-ameigh While this was the edit that Paul Underwood requested in a particular CDK doc, I don't think it makes sense as a global edit in the deploy_steps.adoc file. That would introduce a duplication since we already have this information under "Launch the Quick Start" by default. The CDK content was an exception.

File: .specific/.cdk/deploy_steps.adoc [hunk 1]
```diff
@@ -5,6 +5,12 @@ This Quick Start requires a moderate to high-level understanding of how to manag
 
 The deployment process takes about {deployment_time} to complete, and the template launches in the {default_deployment_region} Region by default.
 
+=== Confirm your AWS account configuration
+
+. Sign in to your AWS account at https://aws.amazon.com with an IAM user role that has the necessary permissions. For details, see link:#_planning_the_deployment[Planning the deployment] earlier in this guide.
+. Make sure that your AWS account is configured correctly, as discussed in the link:#_technical_requirements[Technical requirements] section.
+. Check the AWS Region that's displayed in the upper-right corner of the navigation bar, and change it if necessary. This Region is where the network infrastructure for {partner-product-name} is built. The template is launched in the {default_deployment_region} Region by default. For other choices, see link:#_supported_regions[Supported Regions] earlier in this guide.
```
Comment (MarciaRieferJohnston) [inline .specific/.cdk/deploy_steps.adoc:12]:
```suggestion
. Check the AWS Region that's displayed in the upper-right corner of the navigation bar, and change it if necessary. This Region is where the network infrastructure for {partner-product-name} is built. The template is launched in the {default_deployment_region} Region by default. For more information, see link:#_supported_regions[Supported Regions] earlier in this guide.
```

Comment (MarciaRieferJohnston) [inline .specific/.cdk/deploy_steps.adoc]:
```suggestion
. Check the AWS Region that's displayed in the upper-right corner of the navigation bar, and change it if necessary. This Region is where the network infrastructure for {partner-product-name} is built. The template is launched in the {default_deployment_region} Region by default. For more information, see link:#_supported_aws_regions[Supported AWS Regions] earlier in this guide.
```

File: .specific/.cdk/deploy_steps.adoc [hunk 2]
```diff
@@ -5,6 +5,12 @@ This Quick Start requires a moderate to high-level understanding of how to manag
 
 The deployment process takes about {deployment_time} to complete, and the template launches in the {default_deployment_region} Region by default.
 
+=== Confirm your AWS account configuration
+
+. Sign in to your AWS account at https://aws.amazon.com with an IAM user role that has the necessary permissions. For details, see link:#_planning_the_deployment[Planning the deployment] earlier in this guide.
+. Make sure that your AWS account is configured correctly, as discussed in the link:#_technical_requirements[Technical requirements] section.
+. Check the AWS Region that's displayed in the upper-right corner of the navigation bar, and change it if necessary. This Region is where the network infrastructure for {partner-product-name} is built. The template is launched in the {default_deployment_region} Region by default. For more information, see link:#_supported_regions[Supported Regions] earlier in this guide.
```
Comment (MarciaRieferJohnston) [inline .specific/.cdk/deploy_steps.adoc:12]:
```suggestion
. Check the AWS Region that's displayed in the upper-right corner of the navigation bar, and change it if necessary. This Region is where the network infrastructure for {partner-product-name} is built. The template is launched in the {default_deployment_region} Region by default. For more information, see link:#_supported_aws_regions[Supported AWS Regions] earlier in this guide.
```

**Review by MarciaRieferJohnston — APPROVED** [review]

File: .specific/.cdk/deploy_steps.adoc [hunk 3]
```diff
@@ -5,6 +5,12 @@ This Quick Start requires a moderate to high-level understanding of how to manag
 
 The deployment process takes about {deployment_time} to complete, and the template launches in the {default_deployment_region} Region by default.
 
+=== Confirm your AWS account configuration
+
+. Sign in to your AWS account at https://aws.amazon.com with an IAM user role that has the necessary permissions. For details, see link:#_planning_the_deployment[Planning the deployment] earlier in this guide.
+. Make sure that your AWS account is configured correctly, as discussed in the link:#_technical_requirements[Technical requirements] section.
+. Check the AWS Region that's displayed in the upper-right corner of the navigation bar, and change it if necessary. This Region is where the network infrastructure for {partner-product-name} is built. The template is launched in the {default_deployment_region} Region by default. For more information, see link:#_supported_aws_regions[Supported AWS Regions] earlier in this guide.
```
Comment (MarciaRieferJohnston) [inline .specific/.cdk/deploy_steps.adoc:12]:
```suggestion
. Check the AWS Region that's displayed in the upper-right corner of the navigation bar, and change it if necessary. This Region is where the infrastructure for {partner-product-name} is built. The template is launched in the {default_deployment_region} Region by default. For more information, see link:#_supported_aws_regions[Supported AWS Regions] earlier in this guide.
```

**Review by MarciaRieferJohnston — APPROVED** [review]

**Review by censullo — APPROVED** [review]

**Pull request merged**